e2etest/bgp_test.go

```diff
@@ -955,3 +1030,18 @@ func uint32Ptr(n uint32) *uint32 {
 func boolPtr(b bool) *bool {
 	return &b
 }
+
+func frrIsCommunitySet(cs clientset.Interface, n *frrcontainer.FRR) {
+	exec := executor.ForContainer("frr")
```

Comment:
there's no "frr" container on the host, you need to choose the eBgp or iBgp one. Also, you can use directly n as it implements the `Executor` interface.